docs/tutorials/ssp_profile_catalog_authoring/ssp_profile_catalog_authoring.md

```diff
@@ -4,7 +4,7 @@
 
 Trestle has authoring tools that allow conversion of OSCAL documents to markdown for easy editing - and conversion back to OSCAL for validation and automation.  The author commands are:
 
-1. `catalog-generate` converts a control Catalog to individual controls in markdown format for addition or editing of guidance prose.  `catalog-assemble` then gathers the prose and adds it to the controls in the Catalog.
+1. `catalog-generate` converts a control Catalog to individual controls in markdown format for addition or editing of guidance prose.  `catalog-assemble` then gathers the prose and parameter values and adds them to the controls in the Catalog.
```

Comment:
Rather than "parameter values" should we just say "parameters" as parameters can be defined in Markdown for catalogs and not just setting values as in profiles.

Reply:
Good point.  Changed most of the "parameter values" references to "parameters" unless they were specifically talking about the values themselves, as in moustache replacement